planning/behavior_velocity_intersection_module/src/manager.cpp

```diff
@@ -65,10 +63,12 @@ IntersectionModuleManager::IntersectionModuleManager(rclcpp::Node & node)
     vehicle_info.max_longitudinal_offset_m;
   ip.stuck_vehicle.stuck_vehicle_vel_thr =
     node.declare_parameter<double>(ns + ".stuck_vehicle.stuck_vehicle_vel_thr");
+  /*
   ip.stuck_vehicle.assumed_front_car_decel =
     node.declare_parameter<double>(ns + ".stuck_vehicle.assumed_front_car_decel");
   ip.stuck_vehicle.enable_front_car_decel_prediction =
     node.declare_parameter<bool>(ns + ".stuck_vehicle.enable_front_car_decel_prediction");
+  */
```

Comment:
Do you mean it is still in progress or is it intended?

Reply:
This part is not used as of now, so I commented out temporarily.